Framework/WorkflowAlgorithms/src/AlignAndFocusPowder.cpp

```diff
@@ -434,15 +477,15 @@ void AlignAndFocusPowder::exec() {
   }
 
   // set up a progress bar with the "correct" number of steps
-  m_progress = std::make_unique<Progress>(this, 0., 1., 21);
+  m_progress = std::make_unique<Progress>(this, 0., 1., 22);
```

Comment:
why 22?  any reason for this magic number?

Reply:
There were 21 steps before this functionality, adding this makes one more (22=21+1)